Film view css

MarleneArvedahl opened the pull request:
Added CSS to the page. Tried testing but it failed, continues in the next branch...

MarleneArvedahl committed:
almost all CSS done

MarleneArvedahl committed:
almost all CSS done

MarleneArvedahl committed:
failed test

MarleneArvedahl committed:
failed test

MarleneArvedahl committed:
fixed issues with bookmark

MarleneArvedahl committed:
fixed issues with bookmark

MarleneArvedahl committed:
failed test

MarleneArvedahl committed:
failed test

MarleneArvedahl committed:
fix

MarleneArvedahl committed:
fix

Smustus reviewed (changes requested):
Since there are some failing tests id rather see you adjust them to work or only push the stying :)

Commit:
Merge branch 'main' into filmViewCSS

MarleneArvedahl committed:
test green and navbar is in filmview

Smustus reviewed (changes requested):
Good job! I feel theres some more work to be done on a responsive layout as well as aesthetics on the "close" button. If you want there could also be a test for when a user navigate to the page without a state being sent.

MarleneArvedahl committed:
CSS issue fixed

Commit:
Merge branch 'filmViewCSS' of https://github.com/Robert-Tyborn/video-archives into filmViewCSS

Commit:
Fixed background image and css

Rearragned the html structure, changed css structure, fixed css to filmview to match marlenes figma design. Decided to not have image cover because of how stretched the image looked. Added more media queries to make it more responsive.

Commit:
Merge branch 'main' into filmViewCSS

Commit:
remove black styling from p

In order to prevent filmview to not have white color

Commit:
Merge branch 'main' into filmViewCSS
